### PR TITLE
Ls/653 improve slice creation time

### PR DESCRIPTION
**Pull request by sintinilorenzo**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/server/src/api/common/LibrariesState.ts [hunk 1]
```diff
@@ -10,10 +10,15 @@ const DEFAULT_IMAGE_DIMENSIONS = {
...
 
-export function generateState(env: BackendEnvironment): void {
-  const libraries = (env.manifest.libraries || [])
-    .map((lib) => handleLibraryPath(env.cwd, lib))
-    .filter(Boolean) as ReadonlyArray<Models.Library<Models.Component>>;
+export function generateState(
+  env: BackendEnvironment,
+  libs?: readonly Models.Library<Models.Component>[]
+): void {
+  const libraries =
```
Comment (chengoconnell) [inline packages/slice-machine/server/src/api/common/LibrariesState.ts:17]: 🔥

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "lint-staged": "lint-staged",
     "pretty": "prettier --write .",
     "prettier:check": "prettier --check .",
-    "test": "lerna run test",
+    "test": "lerna run test --no-private",
```
Comment (chengoconnell) [inline package.json:18]: what does this do? :)

**Review by chengoconnell — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (MarcMcIntosh) [reply package.json:18]: Skips the packages we've marked as private. (like the sm-commons and the other ones we don't really maintain anymore :)

**Review by MarcMcIntosh — APPROVED** [review]
Was it all the calls to getEnv that was slowing stuff down?

File: package.json (hunk 2 above)
Comment (chengoconnell) [reply package.json:18]: ah nice thanks for the explanation

**Pull request merged**